packages/chart/src/components/EditorPanel.jsx

```diff
@@ -133,81 +85,6 @@ const Select = memo(({ label, value, options, fieldName, section = null, subsect
   )
 })
 
-const DataSuppression = memo(({ config, updateConfig, data }) => {
```

Comment:
Why is this removed?

Reply:
I saw it wasn't used. I can put it back if we need it. I just saw EditorPanel was 2k lines so wanted to cut where I could.